.github/ISSUE_TEMPLATE/typings_inconsistency.yml

```diff
@@ -0,0 +1,50 @@
+name: "Typings Inconsistency"
+title: "[Typings Inconsistency] "
+labels: [inconsistency]
+assignees: []
+description: "Report inconsistencies between TypeScript's DOM typings and browser behavior."
```

Comment:
I keep being annoyed by usage of DOM in TypeScript, it's really a legacy name that nobody understands. Would be nicer if we call it "web API" here.